news/2025/2025-03-12-skin-of-the-year-2024-results.md

```diff
@@ -0,0 +1,93 @@
+
+The skin itself remains completely loyal to the Windows 11 dark mode theme, with icons, sounds, and gameplay elements all matching closely with the operating system it's based off of. After everybody's Windows updates concluded, **566** community members decided Windosu! is worth their vote, netting Citrusis the #7 spot.
+
+[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/tetos-territory.jpg)](https://osu.ppy.sh/community/forums/topics/1974835)
+
+VOCALOID fans are eating good today, with [Yukoii](https://osu.ppy.sh/users/34419864)'s [Kasane Teto](https://vocaloid.fandom.com/wiki/Kasane_Teto) inspired skin, [Teto's Territory](https://osu.ppy.sh/community/forums/topics/1974835).
+
+Despite the "anime girl" theme, Teto's Territory is actually a surprisingly simplistic skin. Simple, clean elements are present across the entire skin, dominated by pleasant shades of pink and purple, making this not only a good skin to look at, but also to play with. Even though this is Yukoii's first original completed skin, they have received huge praise that ended up earning them **571** votes, and the #6 spot.
+
+[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/yugen-remastered.jpg)](https://osu.ppy.sh/community/forums/topics/1999325)
+
+[Garin](https://osu.ppy.sh/users/2130664) has awoken from their long slumber, and delivered an update to the [original YUGEN skin](https://osu.ppy.sh/community/forums/topics/365036) that we didn't know we needed.
+
+[YUGEN ❯REMASTERED](https://osu.ppy.sh/community/forums/topics/1999325) is as the name states, a remaster of the original skin, while still maintaining the original dim minimalist style that puts all the focus on gameplay, which is why it received so much love back in the day. With new pause and fail screens, new sounds, new mod icons and much more, the community deemed the remaster worthy of **578** votes, landing it in spot #5.
+
+[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/unofficial-deskmat-skin.jpg)](https://osu.ppy.sh/community/forums/topics/2020821)
+
+Another familiar face joins the party! [Cyperdark](https://osu.ppy.sh/users/9893708) has already earned skinning-related accolades by winning last years [Skin of the Year](https://osu.ppy.sh/home/news/2024-02-05-skin-of-the-year-2023-results) contest, so it isn't surprising they are showing up once again.
+
+After last year's release of the [official osu! deskmats](https://osu.ppy.sh/home/news/2024-02-05-skin-of-the-year-2023-results) featuring three different community designs to choose from, the only logical thing to do was combine the designs into one beautiful skin, all while maintaining a clean aesthetic on the gameplay portion. This original idea was rewarded with a whopping **713** votes, and the #4 spot.
```

Comment:
That is the wrong newspost

Reply:
fixed in https://github.com/ppy/osu-wiki/pull/12945/commits/fe590ff1af97bb7b87ef5d85e32b5a5903bf7b6b

Reply:
fixed in https://github.com/ppy/osu-wiki/pull/12945/commits/fe590ff1af97bb7b87ef5d85e32b5a5903bf7b6b